# Translations template for minarca_client.
# Copyright (C) 2023 ORGANIZATION
# This file is distributed under the same license as the minarca_client project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: minarca_client 4.3.0b5.dev1+gd10439c.d20230116\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2023-01-16 10:29-0500\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.10.3\n"

#: minarca_client/main.py:49
#, python-format
msgid "new version %s available"
msgstr ""

#: minarca_client/main.py:51
msgid "fail to check for latest version"
msgstr ""

#: minarca_client/main.py:72
msgid "minarca is already linked, execute `minarca unlink`"
msgstr ""

#: minarca_client/main.py:76
msgid "password: "
msgstr ""

#: minarca_client/main.py:78
msgid "a password is required"
msgstr ""

#: minarca_client/main.py:110
msgid "backup not running"
msgstr ""

#: minarca_client/main.py:129
#, python-format
msgid "Remote server:          %s"
msgstr ""

#: minarca_client/main.py:130
#, python-format
msgid "Connectivity status:    %s"
msgstr ""

#: minarca_client/main.py:131
#, python-format
msgid "Last successful backup: %s"
msgstr ""

#: minarca_client/main.py:131 minarca_client/main.py:132 minarca_client/main.py:133
msgid "Never"
msgstr ""

#: minarca_client/main.py:132
#, python-format
msgid "Last backup date:       %s"
msgstr ""

#: minarca_client/main.py:133
#, python-format
msgid "Last backup status:     %s"
msgstr ""

#: minarca_client/main.py:134
#, python-format
msgid "Details:                %s"
msgstr ""

#: minarca_client/main.py:161
msgid ""
"Minarca manage your computer's backup by linking your computer with a "
"centralized server and running backups on a given schedule."
msgstr ""

#: minarca_client/main.py:180
msgid "start a backup"
msgstr ""

#: minarca_client/main.py:181
msgid "force execution of a backup even if it's not time to run"
msgstr ""

#: minarca_client/main.py:185
msgid "exclude files to be backup"
msgstr ""

#: minarca_client/main.py:186 minarca_client/main.py:192
msgid "file pattern to be exclude. may contains `*` or `?` wildcard"
msgstr ""

#: minarca_client/main.py:191
msgid "include files to be backup"
msgstr ""

#: minarca_client/main.py:197
msgid "link this minarca backup with a minarca server"
msgstr ""

#: minarca_client/main.py:199
msgid "URL to the remote minarca server. e.g.: http://example.com:8080/"
msgstr ""

#: minarca_client/main.py:201
msgid "user name to be used for authentication"
msgstr ""

#: minarca_client/main.py:202
msgid "password to use for authentication. Will prompt if not provided"
msgstr ""

#: minarca_client/main.py:203
msgid "repository name to be used"
msgstr ""

#: minarca_client/main.py:205
msgid "link to remote server even if the repository name already exists"
msgstr ""

#: minarca_client/main.py:210
msgid "list the includes / excludes patterns"
msgstr ""

#: minarca_client/main.py:214
msgid "stop the backup"
msgstr ""

#: minarca_client/main.py:215
msgid "doesn't fail if the backup is not running"
msgstr ""

#: minarca_client/main.py:220
msgid "create required schedule task in crontab or Windows Task Scheduler"
msgstr ""

#: minarca_client/main.py:227
msgid "schedule backup to run hourly"
msgstr ""

#: minarca_client/main.py:230
msgid "schedule backup to run daily"
msgstr ""

#: minarca_client/main.py:237
msgid "schedule backup to run weekly"
msgstr ""

#: minarca_client/main.py:242
msgid "return the current minarca status"
msgstr ""

#: minarca_client/main.py:246
msgid "unlink this minarca client from server"
msgstr ""

#: minarca_client/main.py:250
msgid "open graphical user interface (default when calling minarcaw)"
msgstr ""

#: minarca_client/core/__init__.py:348
#, python-format
msgid "fail to link because repository with name `%s` already exists on remote server"
msgstr ""

#: minarca_client/core/__init__.py:405
msgid "generating identity"
msgstr ""

#: minarca_client/core/__init__.py:411
msgid "exchanging identity with minarca server"
msgstr ""

#: minarca_client/core/__init__.py:415
msgid "generating new identity"
msgstr ""

#: minarca_client/core/__init__.py:419
msgid "exchanging new identity with minarca server"
msgstr ""

#: minarca_client/core/__init__.py:464
#, python-format
msgid "executing command: %s"
msgstr ""

#: minarca_client/core/compat.py:319 minarca_client/ui/templates/home.html:1
#: minarca_client/ui/templates/setup.html:1
msgid "Minarca Backup"
msgstr ""

#: minarca_client/core/compat.py:365
msgid ""
"Keeps your Minarca Backup Software running. If this task is disabled or "
"stopped, your Minarca backup will stop working. This task gets created when "
"linking Minarca with a central backup server. When installing Minarca on a "
"Windows Server, it's preferable to change the settings of this task and "
"select 'Run whether user is logged on or not' to make sure the backup is "
"running even when nobody is using the server."
msgstr ""

#: minarca_client/core/config.py:181
msgid "User's Documents"
msgstr ""

#: minarca_client/core/config.py:188
msgid "Thumbnails cache"
msgstr ""

#: minarca_client/core/config.py:189
msgid "Arrangement of a Windows folder"
msgstr ""

#: minarca_client/core/config.py:190
msgid "Swap System File"
msgstr ""

#: minarca_client/core/config.py:191
msgid "Page System File"
msgstr ""

#: minarca_client/core/config.py:192
msgid "Hibernation System File"
msgstr ""

#: minarca_client/core/config.py:193
msgid "System Volume Information"
msgstr ""

#: minarca_client/core/config.py:194
msgid "System Recovery"
msgstr ""

#: minarca_client/core/config.py:195
msgid "Recycle bin"
msgstr ""

#: minarca_client/core/config.py:196 minarca_client/core/config.py:215
msgid "Temporary Folder"
msgstr ""

#: minarca_client/core/config.py:197
msgid "AutoCAD backup files"
msgstr ""

#: minarca_client/core/config.py:198
msgid "Office temporary files"
msgstr ""

#: minarca_client/core/config.py:199
msgid "Outlook IMAP temporary files"
msgstr ""

#: minarca_client/core/config.py:200
msgid "Outlook POP temporary files"
msgstr ""

#: minarca_client/core/config.py:206
msgid "Desktop Services Store"
msgstr ""

#: minarca_client/core/config.py:212
msgid "dev filesystem"
msgstr ""

#: minarca_client/core/config.py:213
msgid "proc filesystem"
msgstr ""

#: minarca_client/core/config.py:214
msgid "sys filesystem"
msgstr ""

#: minarca_client/core/config.py:216
msgid "Volatile program files"
msgstr ""

#: minarca_client/core/config.py:217
msgid "Mounted filesystems"
msgstr ""

#: minarca_client/core/config.py:218
msgid "External media"
msgstr ""

#: minarca_client/core/config.py:219
msgid "Ext4 Lost and Found"
msgstr ""

#: minarca_client/core/config.py:220
msgid "Hidden temporary files"
msgstr ""

#: minarca_client/core/config.py:221
msgid "Vim Temporary files"
msgstr ""

#: minarca_client/core/exceptions.py:55
#, python-format
msgid ""
"your file specification [%s] doesn't matches any of the base directory of "
"your system"
msgstr ""

#: minarca_client/core/exceptions.py:64
msgid "backup process returned non-zero exit status, check logs for more details"
msgstr ""

#: minarca_client/core/exceptions.py:75
msgid "include patterns are missing"
msgstr ""

#: minarca_client/core/exceptions.py:83
msgid "cannot stop backup when it's not running"
msgstr ""

#: minarca_client/core/exceptions.py:91
msgid "cannot start backup when it's already running"
msgstr ""

#: minarca_client/core/exceptions.py:99
msgid "not configured, use `minarca link` to configure remote host"
msgstr ""

#: minarca_client/core/exceptions.py:107
msgid "backup not yet scheduled to run, you may force execution using `--force`"
msgstr ""

#: minarca_client/core/exceptions.py:117
#, python-format
msgid "cannot establish connection to `%s`, verify if the URL is valid"
msgstr ""

#: minarca_client/core/exceptions.py:128
#, python-format
msgid ""
"the given URL `%s` is not properly formated, verify if the URL is valid. It "
"must start with either https:// or http://"
msgstr ""

#: minarca_client/core/exceptions.py:140
msgid "authentication refused, verify your username and password"
msgstr ""

#: minarca_client/core/exceptions.py:148
msgid "remote server return an error, check remote server log with your administrator"
msgstr ""

#: minarca_client/core/exceptions.py:156
msgid ""
"Unable to connect to the remote server using SSH without password. The "
"problem may be with the remote server. If the problem persists, contact your "
"system administrator to check the SSH server configuration and a possible "
"firewall blocking the connection."
msgstr ""

#: minarca_client/core/latest.py:31
msgid "https://minarca.org/download/"
msgstr ""

#: minarca_client/ui/patterns.py:28
msgid "Included"
msgstr ""

#: minarca_client/ui/patterns.py:28
msgid "Excluded"
msgstr ""

#: minarca_client/ui/patterns.py:80
msgid "Add Folder Pattern"
msgstr ""

#: minarca_client/ui/patterns.py:93
msgid "Add custom pattern"
msgstr ""

#: minarca_client/ui/patterns.py:94
msgid ""
"You may provide a custom pattern to include or exclude file.\n"
"Custom pattern may include wildcard `*` or `?`."
msgstr ""

#: minarca_client/ui/settings.py:56
msgid "New version available"
msgstr ""

#: minarca_client/ui/settings.py:57
#, python-format
msgid ""
"A new version of Minarca (%s) is available from IKUS Software. Do you want to"
" upgrade your copy ?"
msgstr ""

#: minarca_client/ui/settings.py:61
msgid ""
"Minarca automatically checks for new updates. You can change how Minarca "
"checks for updates in Minarca's settings."
msgstr ""

#: minarca_client/ui/settings.py:101
msgid "Are you sure ?"
msgstr ""

#: minarca_client/ui/settings.py:102
msgid "Are you sure you want to disconnect this Minarca agent ?"
msgstr ""

#: minarca_client/ui/settings.py:103
msgid ""
"If you disconnect this computer, this Minarca agent will erase its identity "
"and will no longer run backup on schedule."
msgstr ""

#: minarca_client/ui/setup.py:94 minarca_client/ui/setup.py:95
msgid "Invalid remote server URL !"
msgstr ""

#: minarca_client/ui/setup.py:96
msgid ""
"The remote server URL you entered for the connection is not valid. Check that"
" you have entered the correct value. The URL must begin with `http://` or "
"`https://` followed by a domain name."
msgstr ""

#: minarca_client/ui/setup.py:106 minarca_client/ui/setup.py:107
#: minarca_client/ui/setup.py:133 minarca_client/ui/setup.py:134
msgid "Failed to connect to remote server"
msgstr ""

#: minarca_client/ui/setup.py:108
#, python-format
msgid ""
"Your computer cannot establish a connection to the remote server. Make sure "
"your Internet connection is working and that the following URL is accessible "
"with a Web browser: %s"
msgstr ""

#: minarca_client/ui/setup.py:119 minarca_client/ui/setup.py:120
msgid "Invalid username or password"
msgstr ""

#: minarca_client/ui/setup.py:121
#, python-format
msgid ""
"The username or password you entered to connect to the remote server is not "
"valid.\n"
"\n"
"Details: %s"
msgstr ""

#: minarca_client/ui/setup.py:135
#, python-format
msgid ""
"Your computer cannot establish a connection to the remote server: %s\n"
"\n"
"%s"
msgstr ""

#: minarca_client/ui/setup.py:144 minarca_client/ui/setup.py:145
msgid "Unknown problem when connecting to the remote server"
msgstr ""

#: minarca_client/ui/setup.py:146
#, python-format
msgid ""
"An error occurred during the connection to Minarca server.\n"
"\n"
"Details: %s"
msgstr ""

#: minarca_client/ui/setup.py:160
msgid "Repository name already exists"
msgstr ""

#: minarca_client/ui/setup.py:161
msgid "Do you want to replace the existing repository ?"
msgstr ""

#: minarca_client/ui/setup.py:162
msgid ""
"The repository name you have entered already exists on the remote server. If "
"you continue with this repository, you will replace it's content using this "
"computer. Otherwise, you must enter a different repository name."
msgstr ""

#: minarca_client/ui/status.py:55
#, python-format
msgid "Welcome %s"
msgstr ""

#: minarca_client/ui/status.py:64
msgid "Backup is healthy"
msgstr ""

#: minarca_client/ui/status.py:66
msgid "Backup failed"
msgstr ""

#: minarca_client/ui/status.py:68
msgid "Backup in progress"
msgstr ""

#: minarca_client/ui/status.py:70
msgid "Backup is stale"
msgstr ""

#: minarca_client/ui/status.py:72
msgid "Backup was interrupted"
msgstr ""

#: minarca_client/ui/status.py:74
msgid "No backup yet"
msgstr ""

#: minarca_client/ui/status.py:76
msgid "Backup is not healthy"
msgstr ""

#: minarca_client/ui/status.py:92
msgid "Last backup failed"
msgstr ""

#: minarca_client/ui/status.py:94
msgid "Currently running"
msgstr ""

#: minarca_client/ui/status.py:96
msgid "Stale"
msgstr ""

#: minarca_client/ui/status.py:98
msgid "Interrupted"
msgstr ""

#: minarca_client/ui/status.py:99
msgid "Unknown"
msgstr ""

#: minarca_client/ui/status.py:107
#, python-format
msgid "Complete successfully on %s. No background jobs using system resources."
msgstr ""

#: minarca_client/ui/status.py:109
#, python-format
msgid ""
"Last backup failed on %s for the following reason: %s\n"
"No background jobs using system resources."
msgstr ""

#: minarca_client/ui/status.py:113
msgid "Backup is currently running in background and using system resources."
msgstr ""

#: minarca_client/ui/status.py:115
#, python-format
msgid ""
"Was started in background on %s, but is currently stale an may use system "
"resources."
msgstr ""

#: minarca_client/ui/status.py:118
#, python-format
msgid ""
"Was interrupted on %s. May be caused by loss of connection, computer standby "
"or manual interruption.\n"
"No background jobs using system resources."
msgstr ""

#: minarca_client/ui/status.py:124
msgid ""
"Initial backup need to be started. You may take time to configure your "
"parameters and start your initial backup manually.\n"
"No background jobs using system resources."
msgstr ""

#: minarca_client/ui/status.py:141
msgid "Stop backup"
msgstr ""

#: minarca_client/ui/status.py:142
msgid "Start backup"
msgstr ""

#: minarca_client/ui/status.py:181 minarca_client/ui/status.py:182
msgid "Fail to start backup !"
msgstr ""

#: minarca_client/ui/status.py:183
msgid ""
"A fatal error occurred when trying to start the backup process. This usually "
"indicate a problem with the installation. Try re-installing Minarca Backup."
msgstr ""

#: minarca_client/ui/templates/home.html:7
msgid "Home"
msgstr ""

#: minarca_client/ui/templates/home.html:10
#: minarca_client/ui/templates/patterns.html:3
msgid "Select files"
msgstr ""

#: minarca_client/ui/templates/home.html:14
#: minarca_client/ui/templates/schedule.html:4
msgid "Schedule backup"
msgstr ""

#: minarca_client/ui/templates/home.html:18
#: minarca_client/ui/templates/settings.html:4
msgid "Settings"
msgstr ""

#: minarca_client/ui/templates/home.html:22
msgid "Help"
msgstr ""

#: minarca_client/ui/templates/patterns.html:6
msgid "Add file"
msgstr ""

#: minarca_client/ui/templates/patterns.html:7
msgid "Add folder"
msgstr ""

#: minarca_client/ui/templates/patterns.html:8
msgid "Add ..."
msgstr ""

#: minarca_client/ui/templates/patterns.html:14
msgid "Remove pattern"
msgstr ""

#: minarca_client/ui/templates/schedule.html:7
msgid "Select how often you want your backup to be performed."
msgstr ""

#: minarca_client/ui/templates/schedule.html:10
msgid "Frequency"
msgstr ""

#: minarca_client/ui/templates/schedule.html:11
msgid "Hourly"
msgstr ""

#: minarca_client/ui/templates/schedule.html:12
msgid "Daily"
msgstr ""

#: minarca_client/ui/templates/schedule.html:13
msgid "Weekly"
msgstr ""

#: minarca_client/ui/templates/schedule.html:14
msgid "Monthly"
msgstr ""

#: minarca_client/ui/templates/settings.html:9
msgid "Updates"
msgstr ""

#: minarca_client/ui/templates/settings.html:10
msgid "Check now"
msgstr ""

#: minarca_client/ui/templates/settings.html:14
msgid "Check for update at startup"
msgstr ""

#: minarca_client/ui/templates/settings.html:18
msgid "Checking for updates"
msgstr ""

#: minarca_client/ui/templates/settings.html:20
msgid "Current version is up-to-date"
msgstr ""

#: minarca_client/ui/templates/settings.html:22
msgid "Your version may be upgraded"
msgstr ""

#: minarca_client/ui/templates/settings.html:24
msgid "Cannot check for updates. Try again later."
msgstr ""

#: minarca_client/ui/templates/settings.html:36
msgid "Disconnect the backup"
msgstr ""

#: minarca_client/ui/templates/settings.html:37
msgid "Disconnect"
msgstr ""

#: minarca_client/ui/templates/settings.html:39
msgid "When you disconnect this computer, the backup process will no longer work."
msgstr ""

#: minarca_client/ui/templates/setup.html:5
msgid "Data Backup"
msgstr ""

#: minarca_client/ui/templates/setup.html:10
msgid "Log me in"
msgstr ""

#: minarca_client/ui/templates/setup.html:18
msgid "Remote Minarca server"
msgstr ""

#: minarca_client/ui/templates/setup.html:21
msgid "e.g.: https://www.servername.com"
msgstr ""

#: minarca_client/ui/templates/setup.html:24
msgid "Username"
msgstr ""

#: minarca_client/ui/templates/setup.html:29
msgid "Password"
msgstr ""

#: minarca_client/ui/templates/setup.html:32
msgid "If two-factor authentication is enabled on your account, use an Access Token."
msgstr ""

#: minarca_client/ui/templates/setup.html:36
msgid "Repository name"
msgstr ""

#: minarca_client/ui/templates/setup.html:42
msgid "Sign in"
msgstr ""

#: minarca_client/ui/templates/setup.html:45
msgid "Connecting"
msgstr ""

#: minarca_client/ui/templates/setup.html:49
msgid ""
"To configure Minarca to backup your data, you must provide URL to Minarca "
"server. This information should be provided by your system administrator."
msgstr ""

#: minarca_client/ui/templates/status.html:7
msgid "View my account online"
msgstr ""

#: minarca_client/ui/templates/status.html:18
msgid "Backup Status: "
msgstr ""

#: minarca_client/ui/templates/status.html:26
msgid "Last Backup: "
msgstr ""

